### PR TITLE
Added median and min to performance report.

### PR DESCRIPTION
**Pull request by pcicotti**

### CODE REVIEW

File: tftrt/examples/image-classification/image_classification.py [hunk 1]
```diff
@@ -63,13 +63,16 @@ def __init__(self, target_duration=None, iteration_limit=None):
     def before_run(self, run_context):
         if not self.start_time:
             self.start_time = time.time()
-            print("    running for target duration from %d", self.start_time)
+            if self.target_duration:
+                print("    running for target duration {} seconds".format(self.target_duration))
+                print(" from {}".format(time.asctime(time.localtime(self.start_time))))
```
Comment (pooyadavoodi) [inline tftrt/examples/image-classification/image_classification.py]: I think it's better to print these two on the same line.

File: tftrt/examples/image-classification/image_classification.py [hunk 2]
```diff
@@ -63,13 +63,16 @@ def __init__(self, target_duration=None, iteration_limit=None):
...
 
     def after_run(self, run_context, run_values):
         if self.target_duration:
             current_time = time.time()
             if (current_time - self.start_time) > self.target_duration:
-                print("    target duration %d reached at %d, requesting stop" % (self.target_duration, current_time))
+                print("    target duration {}".format(self.target_duration))
```
Comment (pooyadavoodi) [inline tftrt/examples/image-classification/image_classification.py]: Also here, I think it's better to print these two on the same line.

File: tftrt/examples/image-classification/image_classification.py (hunk 1 above)
Comment (pcicotti) [reply tftrt/examples/image-classification/image_classification.py]: you are correct. I meant to split only the code over two lines, not the message.

File: tftrt/examples/image-classification/image_classification.py (hunk 2 above)
Comment (pcicotti) [reply tftrt/examples/image-classification/image_classification.py]: same as above

**Comment by pooyadavoodi** [comment]
Thanks @pcicotti for the PR.

**Pull request merged**